snmp/datadog_checks/snmp/data/default_profiles/chatsworth_pdu.yaml

```diff
@@ -27,396 +33,396 @@ metric_tags:
 metrics:
   ### [Legacy] PDU
   - # [legacy] This table contains a list of PDU entries that are being monitored by the PDU.
-    MIB: CPI-UNITY-MIB
+    MIB: CPI-LEGACY-MIB
```

Comment:
I think we should keep the original here in the public profile since it might be mis-leading for users since `CPI-LEGACY-MIB` does not exist elsewhere.

Reply:
Maybe you can change the MIB when generating snmprec and creating e2e test, but revert the change in the profile after.

Reply:
Yes it sounds fair :)